superset/views/css_templates.py

```diff
@@ -54,8 +48,15 @@ def list(self) -> FlaskResponse:
 class CssTemplateAsyncModelView(  # pylint: disable=too-many-ancestors
     CssTemplateModelView
 ):
-    include_route_methods = {RouteMethod.API_READ}
+    include_route_methods = RouteMethod.API_READ
     class_permission_name = "CssTemplate"
     method_permission_name = MODEL_VIEW_RW_METHOD_PERMISSION_MAP
 
     list_columns = ["template_name", "css"]
+
+    @expose_api(name="read", url="/api/read", methods=["GET"])
+    @has_access_api
+    @permission_name("list")
+    @deprecated(eol_version="5.0.0")
```

Comment:
Could we just remove this? not used on the frontend and a REST API exists for this for some time now

Reply:
@dpgaspar We discussed this during the release meeting and we agreed to remove these legacy endpoints during a breaking window even if they are not under `v1`. That's what we did in the past and we just want to be mindful that someone out there might still be using the endpoint.